sim_pipeline/gg_lens.py

```diff
@@ -10,7 +10,7 @@ class GGLens(object):
     class to manage individual galaxy-galaxy lenses
     """
 
-    def __init__(self, source_dict, deflector_dict, cosmo):
+    def __init__(self, source_dict, deflector_dict, cosmo, test_area=90 * np.pi):
```

Comment:
can you specify what 'test_area' means here?

Reply:
meaning the area of position around lensing galaxies that source galaxies inside the position need to be tested.

Reply:
so that the source galaxies' position can be generate within the area.